nginx-default.conf

```diff
@@ -0,0 +1,44 @@
+server {
+    listen       80;
+    server_name  localhost;
+
+    #access_log  /var/log/nginx/host.access.log  main;
+
+    location / {
+        root   /usr/share/nginx/html;
+        index  index.html index.htm;
+        try_files $uri $uri/ /index.html /index.htm
```

Comment:
I still think you need a semi-colon here?
This works for me now when I add it in

Reply:
You are correct.